alibi_detect/utils/keops/tests/test_kernels_keops.py

```diff
@@ -64,3 +63,59 @@ def test_gaussian_kernel(gaussian_kernel_params):
             k_xx_argmax, k_xx_min, k_xy_min = k_xx_argmax[0], k_xx_min[0], k_xy_min[0]
         assert (torch.arange(n_instances[0]) == k_xx_argmax.cpu().view(-1)).all()
         assert (k_xx_min >= 0.).all() and (k_xy_min >= 0.).all()
+
+
+if has_keops:
+    class MyKernel(nn.Module):
+        def __init__(self):
+            super().__init__()
+
+        def forward(self, x: LazyTensor, y: LazyTensor) -> LazyTensor:
+            return (- ((x - y) ** 2).sum(-1)).exp()
```

Comment:
Is typing the only reason why this needs to be inside `if has_keops` block? If so we could just use forward-references to `'LazyTensor'`? @ascillitoe

Reply:
Good point. Looks like that might be the case...

Reply:
Did the foward-ref not work?

Reply:
@jklaise and I were chatting earlier and decided (correct me if I'm wrong) that it's not necessarily a better solution, just a bit different, so I kept it as is.

Reply:
OK fair enough!